adapters/adnuntius/adnuntius.go

```diff
@@ -458,13 +459,6 @@ func generateAdResponse(ad Ad, imp openrtb2.Imp, html string, request *openrtb2.
 		}}
 	}
 
-	adDomain := []string{}
-	for _, url := range ad.DestinationUrls {
```

Comment:
It seems like `destinationUrls` is getting replaced by `advertiserDomains`. Do you foresee `destinationUrls` to still be used for any other purpose in the future? If not, could we remove the `destinationUrls` instances from the JSON files?

Reply:
@antosarho please see above.

Reply:
Sorry for the late reply -- I thought this changeset was already merged because I got it confused with another pull request.

But `destinationUrls` is not being replaced by `advertiserDomains`. `destinationUrls` will remain and continue to perform its function as providing a list of complete URLs that belong to the ad.

`advertiserDomains` is instead a dedicated new field that the Adnuntius ad server has introduced that is meant specifically for the adomain ortb field.

Reply:
By the way, I didn't make any code changes since you last took a look -- I just rebased the changeset and pushed.